doc/source/merging.rst

```diff
@@ -86,12 +86,18 @@ some configurable handling of "what to do with the other axes":
   it is passed, in which case the values will be selected (see below). Any None
   objects will be dropped silently unless they are all None in which case a
   ValueError will be raised.
-- ``axis`` : {0, 1, ...}, default 0. The axis to concatenate along.
+- ``axis`` : {0, 1, ...}, default 0. The axis to concatenate along. 0 is the index
+  axis, 1 the column axis etc.
 - ``join`` : {'inner', 'outer'}, default 'outer'. How to handle indexes on
   other axis(es). Outer for union and inner for intersection.
 - ``ignore_index`` : boolean, default False. If True, do not use the index
-  values on the concatenation axis. The resulting axis will be labeled 0, ...,
-  n - 1. This is useful if you are concatenating objects where the
+  values on the concatenation axis. If the concatenation axis is 0, it is the
```

Comment:
this is too much text

Reply:
What makes you think that? For me there is a great difference between an index column and column names so that I wanted to stress this. How would you want to shorten it?

Reply:
you have a bunch of text here
it is too dense to read